.env.example

```diff
@@ -20,6 +20,7 @@ REACT_APP_STYLE_TYPE=green-accent
 REACT_APP_CONTEXT=webapp
 REACT_APP_SHOW_AUTH_BUTTON=true
 REACT_APP_LOGOUT_PAGE_URL=http://localhost:8080/conda-store/logout?next=/
+REACT_APP_ROUTER_TYPE=browser
```

Comment:
Since you are adding a new configuration, would it be worth updating the docs?  https://conda.store/conda-store-ui/how-tos/configure-ui

Reply:
Yes, if we're happy with this, I will update the docs

Reply:
Seems fine, it would appear that the `memory` setting would _only_ be needed for the JLab extension right?

Reply:
Ok I will approve this, but will need an update on the docs.